docker/standalone/README.md

```diff
@@ -7,8 +7,16 @@ If you're using a different operating system, you can follow the official Docker
 
 Once you have Docker installed, you can run CtrlPanel standalone Docker by executing the following command:
 
+Recommended way via Docker Compose:
+
+1. Copy and configure your docker compose file to your needs `curl -L -o compose.yaml https://raw.githubusercontent.com/Ctrlpanel-gg/panel/blob/main/docker/standalone/compose.yaml`.
+2. Create the env file in the same directory as the compose file `touch env_file`.
+3. When installing you need to update the `env_file` file. Change those two variables to: `MEMCACHED_HOST=redis` and `REDIS_HOST=redis`, to use the Redis server which comes with the docker compose installation.
+
+Running as commandline command:
+
 ```bash
-docker run -p 80:80 -p 443:443 -v /path/to/website_files:/var/www/html -v /path/to/nginx_config:/etc/nginx/conf.d/ ctrlpanel/ctrlpanel
+docker run -p 80:80 -p 443:443 -v /path/to/website_files:/var/www/html -v /path/to/nginx_config:/etc/nginx/conf.d/ ghcr.io/ctrlpanel-gg/panel:latest
 ```
```

Comment:
here I thin that you should reverse the two, since the docker run command is the most used :D

Reply:
I dont understand what you mean. i just changed this so it will use the docker image which will be available on the github container registry. the `ctrlpanel/ctrlpanel` does not exist anywhere. Please elaborate what you mean by this.

Reply:
No I just mean the two section put the docker command above the docker compose